docs/pages/setup/admin/adding-nodes.mdx

```diff
@@ -98,18 +93,20 @@ $ tctl tokens ls
 # Token                            Type            Expiry Time
 # ------------------------         -----------     ---------------
 # (=presets.tokens.first=)         Node            25 Sep 18 00:21 UTC
-
-# ... or revoke an invite token before it's used:
-$ tctl tokens rm (=presets.tokens.first=)
 ```
 
-If you want to provide your own token, you can do so using the `--token` flag:
+<Details title="Tokens with known values">
```

Comment:
Nit: Maybe say "Provide your own token value" as a title instead or something like this, and add a warning to use cryptographically secure token values for better security.